src/layouts/Layout.astro

```diff
@@ -7,7 +7,7 @@ import Footer from "../components/Footer/Footer.jsx";
 const { title, pathName } = Astro.props;
 
 const description =
-	"Hack the Hill, the organization behind Canada's Capital's premier flagship hackathon event & more. This Fall 2024, come prepared to network and compete with the best at Canada's Capital's largest hackathon!";
+	"Capital Technology Network, the organization behind Canada's Capital's premier flagship hackathon event & more. This Fall 2024, come prepared to network and compete with the best at Canada's Capital's largest hackathon!";
```

Comment:
I feel like it would be good to mention "Hack the Hill" in the 2nd portion of this sentence
"...the organization behind Hack the Hill, Canada's Capital's hackathon..."

Reply:
Also it should say "The Capital Technology Network" not "Capital Technology Network"